--- /home/sergeyg/.emacs.d/elpa/window-purpose-1.4/window-purpose-x.el	2015-08-24 00:14:32.623390902 +0200
+++ /home/sergeyg/tmp/window-purpose-x.el	2015-08-23 22:45:22.097989437 +0200
@@ -60,15 +60,10 @@
                (0.0 0.0 0.19333333333333333 0.5))
      (:purpose buffers :purpose-dedicated t :width 0.16 :height 0.4722222222222222 :edges
                (0.0 0.5 0.19333333333333333 0.9722222222222222)))
-    (t
-     (29 0 125 35)
-    (:purpose edit :purpose-dedicated t :width 0.6 :height 0.85 :edges
-              (0.19333333333333333 0.0 0.8266666666666667 0.85))
-    (:purpose misc :purpose-dedicated t :width 0.6 :height 0.1 :edges
-              (0.19333333333333333 0.8722222222222222 0.8266666666666667 0.9722222222222222))
-     )
+    (:purpose edit :purpose-dedicated t :width 0.6 :height 0.9722222222222222 :edges
+              (0.19333333333333333 0.0 0.8266666666666667 0.9722222222222222))
     (:purpose ilist :purpose-dedicated t :width 0.15333333333333332 :height 0.9722222222222222 :edges
-              (0.8266n666666666667 0.0 1.0133333333333334 0.9722222222222222)))
+              (0.8266666666666667 0.0 1.0133333333333334 0.9722222222222222)))
   "Window layout for purpose-x-code1-dired-ibuffer.
 Has a main 'edit window, and two side windows - 'dired and 'buffers.
 All windows are purpose-dedicated.")
@@ -79,10 +74,7 @@
                 :mode-purposes
                 '((ibuffer-mode . buffers)
                   (dired-mode . dired)
-                  (imenu-list-major-mode . ilist)
-                  (inferior-python-mode . misc)
-                  (python-inferior-mode . misc)
-                  )))
+                  (imenu-list-major-mode . ilist))))
 
 (defvar purpose-x-code1-buffers-changed nil
   "Internal variable for use with `frame-or-buffer-changed-p'.")
@@ -169,8 +161,7 @@
   (imenu-list-minor-mode)
   (frame-or-buffer-changed-p 'purpose-x-code1-buffers-changed)
   (add-hook 'post-command-hook #'purpose-x-code1-update-changed)
-  (purpose-set-window-layout purpose-x-code1--window-layout)
-)
+  (purpose-set-window-layout purpose-x-code1--window-layout))
 
 (defun purpose-x-code1-unset ()
   "Unset purpose-x-code1."
